src/org/zaproxy/zap/extension/alertFilters/ContextAlertFilterPanel.java

```diff
@@ -103,7 +104,11 @@ public AlertFiltersMultipleOptionsPanel(ExtensionAlertFilters extension, AlertFi
 			super(model);
 			this.extension = extension;
 
-			getTable().getColumnExt(0).setMaxWidth(70);
+			if (getTable().getColumnExt(0).getHeaderRenderer()==null) {// If there isn't a header renderer then setMaxWidth
+				Component c = getTable().getTableHeader().getDefaultRenderer().getTableCellRendererComponent(null, getTable().getColumnExt(0).getHeaderValue(), false, false, 0, 0);
+				getTable().getColumnExt(0).setMaxWidth(c.getMaximumSize().width);
```

Comment:
The custom renderer is obtained with the call "getTable().getColumnExt(0).getHeaderRenderer()", if it's non-null then it's using a custom renderer and we should use that.